pkg/config/config.go

```diff
@@ -1,14 +1,15 @@
 package config
 
 import (
+	"encoding/json"
 	"fmt"
 	"os"
-	"path/filepath"
 	"sort"
 	"strings"
 
 	"github.com/kirsle/configdir"
 	"github.com/spf13/viper"
+	"gopkg.in/yaml.v3"
```

Comment:
We use github.com/goccy/go-yaml not this one in Grizzly.

Reply:
hmm, it's not in the go.mod file

Reply:
Ahh right, that's in an unmerged branch. Ignore this comment then.